src/plugins/intel_npu/src/al/include/intel_npu/config/npuw.hpp

```diff
@@ -76,10 +76,55 @@ DEFINE_OPT(NPUW_LLM_OPTIMIZE_V_TENSORS, bool, true, npuw::llm::optimize_v_tensor
 
 namespace npuw {
 namespace llm {
+enum class PrefillHint { DYNAMIC, STATIC };
 enum class GenerateHint { FAST_COMPILE, BEST_PERF };
 }  // namespace llm
 }  // namespace npuw
 
+struct NPUW_LLM_PREFILL_HINT final : OptionBase<NPUW_LLM_PREFILL_HINT, ::intel_npu::npuw::llm::PrefillHint> {
```

Comment:
Won't it work via `DEFINE_OPT`? Or you explicitly need `parse` and `to_string`?

Reply:
I don't really know, but GENERATE_HINT was implemented the same way for some reason.
I don't like the approach, it's too boilerplate, but hopefully we can sort it out later